roles/openshift_hosted/tasks/router/router.yml

```diff
@@ -1,7 +1,9 @@
 ---
 - fail:
```

Comment:
Not related to this PR, but it might be good to switch to use assert for these use cases rather than fail.